hadoop-ozone/dist/src/shell/conf/log4j.properties

```diff
@@ -31,21 +31,40 @@ log4j.appender.NullAppender=org.apache.log4j.varia.NullAppender
 #
 # Rolling File Appender - cap space usage at 5gb.
 #
-hadoop.log.maxfilesize=256MB
-hadoop.log.maxbackupindex=20
-log4j.appender.RFA=org.apache.log4j.RollingFileAppender
-log4j.appender.RFA.File=${hadoop.log.dir}/${hadoop.log.file}
+#hadoop.log.maxfilesize=256MB
+#hadoop.log.maxbackupindex=20
+#log4j.appender.RFA=org.apache.log4j.RollingFileAppender
+#log4j.appender.RFA.File=${hadoop.log.dir}/${hadoop.log.file}
 
-log4j.appender.RFA.MaxFileSize=${hadoop.log.maxfilesize}
-log4j.appender.RFA.MaxBackupIndex=${hadoop.log.maxbackupindex}
+#log4j.appender.RFA.MaxFileSize=${hadoop.log.maxfilesize}
+#log4j.appender.RFA.MaxBackupIndex=${hadoop.log.maxbackupindex}
 
-log4j.appender.RFA.layout=org.apache.log4j.PatternLayout
+#log4j.appender.RFA.layout=org.apache.log4j.PatternLayout
 
 # Pattern format: Date LogLevel LoggerName LogMessage
-log4j.appender.RFA.layout.ConversionPattern=%d{ISO8601} [%t] %p %c: %m%n
+#log4j.appender.RFA.layout.ConversionPattern=%d{ISO8601} [%t] %p %c: %m%n
 # Debugging Pattern format
 #log4j.appender.RFA.layout.ConversionPattern=%d{ISO8601} [%t] %-5p %c{2} (%F:%M(%L)) - %m%n
 
+#
+# Rolling File Appender that gzip rolled over log files
+#
+
+log4j.appender.RFA=org.apache.log4j.rolling.RollingFileAppender
+log4j.appender.RFA.rollingPolicy=org.apache.log4j.rolling.FixedWindowRollingPolicy
+log4j.appender.RFA.triggeringPolicy=org.apache.log4j.rolling.SizeBasedTriggeringPolicy
+
+log4j.appender.RFA.rollingPolicy.ActiveFileName=${hadoop.log.dir}/${hadoop.log.file}
+log4j.appender.RFA.rollingPolicy.FileNamePattern=${hadoop.log.dir}/${hadoop.log.file}-%i.log.gz
```

Comment:
This will result in archive files named like `hadoop.log-1.log.gz`, so `.log` is duplicated.